safari/environment.js

```diff
@@ -48,5 +48,8 @@ addInterceptor('deleteCookies', cookies => {
 // The iframe hack doesn't work anymore, so Safari has no way to add urls to history
 addInterceptor('addURLToHistory', () => {});
 
+// XXX No point in implementing this when addURLToHistory is not implemented
+addInterceptor('isURLVisited', () => undefined);
```

Comment:
`false` instead of `undefined` (so minor, I know, sorry)

I guess `undefined` would represent "indeterminate", but I don't really want to allow the possibility of `if (visited === false) ... else if (visited === undefined)` (prefer to throw if we need to handle an unimplemented API, probably).